lua/sim/units/ACUUnit.lua

```
@@ -139,6 +263,82 @@ ACUUnit = ClassUnit(CommandUnit) {
+                    and (not t.damageType or t.damageType == damageType)
+                then
+                    deathMessage = table.random(t.strings)
+                    if t.useInistigatorName then
+                        local customName = instigator.CustomName
+                        if not customName then
+                            local ainames = import('/lua/ui/lobby/aiNames.lua').ainames[string.lower(self.Blueprint.FactionCategory)]
+                            if ainames then
+                                customName = tostring(table.random(ainames))
+                            else
+                                continue
+                            end
```

Luacheck (1.2.0)

[error] expected '=' near 'end'

(E011)